tests/core/test_hugepage.py

```diff
@@ -37,8 +37,9 @@ def test_hugepage_support(self):
         self.fstab.Fstab().get_entry_by_attr.return_value = 'old fstab entry'
         self.fstab.Fstab().Entry.return_value = 'new fstab entry'
         hugepage.hugepage_support('nova')
-        sysctl_expect = ("{vm.hugetlb_shm_group: '1010', "
-                         "vm.max_map_count: 65536, vm.nr_hugepages: 256}\n")
+        sysctl_expect = ("vm.hugetlb_shm_group: '1010'\n"
+                         "vm.max_map_count: 65536\n"
+                         "vm.nr_hugepages: 256\n")
```

Comment:
Maybe use a multi-line string literal? It looks like a tuple for someone skim-reading.